### PR TITLE
Fix/module builder sprout4

### PR DESCRIPTION
**Pull request by jamiemonksuk**
Quick fix to get modules processing properly for dbtools module/model builder in sprout4

### CODE REVIEW

File: src/sprout/Controllers/DbToolsController.php [hunk 1]
```diff
@@ -2281,7 +2280,8 @@ public function moduleBuilderExistingUploadAction()
             if (!preg_match('!^[a-zA-Z0-9]+$!', $_POST['existing'])) {
```
Comment (gwillz) [inline src/sprout/Controllers/DbToolsController.php:2280]: we could probably replace this funky validation here with a `$module === null`.

Comment (jamiemonksuk) [reply src/sprout/Controllers/DbToolsController.php]: I pushed something for that

**Comment by jamiemonksuk** [comment]
Bump :D

**Review by gwillz — APPROVED** [review]
Can't complain, all good things!

**Pull request merged**